bread/templates/bread/includes/browse.html

```diff
@@ -9,28 +9,41 @@
    {% if last_url %}<a href="{{ last_url }}">[{% trans "Last" %}]</a>{% endif %}
 {% endif %}
 
- {% if filter != None and filter.form != None %}
+ {% if has_filter or has_search %}
    <form method="GET">
-       {% if filter.form.non_field_errors %}
-         <div class="control-group">
-           {% for err in filter.form.non_field_errors %}
-             <div class="alert alert-error help-block">
-               {{ err }}
-             </div>
-           {% endfor %}
-         </div>
-       {% endif %}
+     {% if has_search %}
+       <div class="control-group">
+         <p>{% blocktrans with names=view.bread.verbose_name_plural %}Search {{ names }}{% endblocktrans %}</p>
```

Comment:
`verbose_name_plural` should be in the template context, so this should be `names=verbose_name_plural`.